Mathlib/RingTheory/Coalgebra/Basic.lean

```diff
@@ -80,7 +105,19 @@ theorem rTensor_counit_comul (a : A) : counit.rTensor A (comul a) = 1 ⊗ₜ[R]
 theorem lTensor_counit_comul (a : A) : counit.lTensor A (comul a) = a ⊗ₜ[R] 1 :=
   LinearMap.congr_fun lTensor_counit_comp_comul a
 
+open BigOperators CoalgebraStruct
+
+lemma sum_counit_tmul_eq {a : A} (repr : Repr R a) :
+    ∑ i ∈ repr.index, counit (R := R) (repr.left i) ⊗ₜ (repr.right i) = 1 ⊗ₜ[R] a := by
+  simpa [← repr.eq, map_sum] using congr($(rTensor_counit_comp_comul (R := R) (A := A)) a)
+
+lemma sum_tmul_counit_eq {a : A} (repr : Repr R a) :
+    ∑ i ∈ repr.index, (repr.left i) ⊗ₜ counit (R := R) (repr.right i) = a ⊗ₜ[R] 1 := by
+  simpa [← repr.eq, map_sum] using congr($(lTensor_counit_comp_comul (R := R) (A := A)) a)
```

Comment:
Now the `repr` is bundled, would these be good `simp` lemmas?

Reply:
added `simp` tag